### PR TITLE
ci(aio): use correct URL for preview in PWA score test

### PR DESCRIPTION
**Pull request by gkalpak**
In 4268c82, the preview URLs were changed to not use the whole SHA, but just the 7 first characters.

### CODE REVIEW

File: aio/scripts/deploy-preview.sh [hunk 1]
```diff
@@ -7,9 +7,11 @@ exec 3>&1
 
 readonly INPUT_DIR=dist/
 readonly OUTPUT_FILE=/tmp/snapshot.tar.gz
-readonly AIO_BUILDS_DOMAIN=ngbuilds.io
+readonly AIO_BUILDS_DOMAIN=ngbuilds.ioq
```
Comment (petebacondarwin) [inline aio/scripts/deploy-preview.sh]: is this change intentional?

Comment (gkalpak) [inline aio/scripts/deploy-preview.sh]: Nope :)

**Review by petebacondarwin — APPROVED** [review]
LGTM

**Comment by petebacondarwin** [comment]
Cherry picked to 4.2.x via 1a476b81ab32bad8ce92740e705fe0ce6f4e71fe
